app/templates/components/public/speaker-item.hbs

```diff
@@ -48,19 +48,21 @@
             {{#if session.startsAt}}
               {{moment-format session.startsAt 'DD MMM YY'}}
             {{/if}}
-            <br>
-            <span class="session-location">
-              <strong>
-                {{session.microlocation.name}}
-              </strong>
-            </span>
+            {{#if session.microlocation}}
+              <br>
+              <span class="session-location">
+                <strong>
+                  {{concat (t 'Microlocation - ') session.microlocation.name}}
```

Comment:
There is not need to highlight `microlocation` , It is looking odd
<img width="502" alt="Screenshot 2019-06-10 at 11 01 10 AM" src="https://user-images.githubusercontent.com/44091822/59174460-16b37080-8b6f-11e9-831e-84d1970467f6.png">

Reply:
Like I said earlier this is what was implemented earlier intentionally

Reply:
@shreyanshdwivedi I am saying previously before the location , The description was not given , `MicroLocation - XYZ` type

Reply:
I think when an outsider will look at this, `LHC` or `XYZ` will make less sense to him unless he know the place well. So I just added `Microlocation` so that it is clear that it represents a microlocation. 
However if the mentors deny to add it, I'll remove it